index.js

```diff
@@ -0,0 +1,30 @@
+const taskInput = document.getElementById('myInput');
+const taskList = document.getElementById('taskList');
+
+taskInput.addEventListener('keydown', function (event) {
+  const taskText = taskInput.value.trim();
+  if (event.key === 'Enter' && taskText !== '') {
+    const taskItem = createTaskItem(taskText);
+    taskList.appendChild(taskItem);
+    taskInput.value = '';
+  }
+});
+
+function createTaskItem(text) {
+  console.log(text);
+  const taskItem = document.createElement('li');
+  const taskText = document.createElement('span');
+  const deleteButton = document.createElement('img');
```

Comment:
DOM을 다루다보니 어떤 element를 만들었는지 아래쪽에서 헷갈릴 수 있어서 변수명에 접미사로 element의 정보를 붙여줘도 좋을 것 같아요~

```suggestion
  const taskItemLi = document.createElement('li');
  const taskTextSpan = document.createElement('span');
  const deleteButtonImg = document.createElement('img');
```

Reply:
넵 수정했습니다!